optaweb-vehicle-routing-frontend/package.json

```diff
@@ -2,13 +2,17 @@
   "private": true,
   "license": "Apache-2.0",
   "dependencies": {
-    "@patternfly/react-core": "^1.36.0",
-    "@patternfly/react-icons": "^2.9.2",
-    "leaflet": "^1.3.4",
-    "react": "^16.6.3",
-    "react-dom": "^16.6.3",
-    "react-leaflet": "^2.1.2",
-    "react-redux": "^5.1.1",
+    "@patternfly/patternfly-next": "^1.0.175",
```

Comment:
This package is [deprecated](https://www.npmjs.com/package/@patternfly/patternfly-next). Please use `@patternfly/patternfly`.

Reply:
replaced deprecated dependency